README.md

```diff
@@ -11,66 +11,56 @@ FitBenchmarking is a tool for comparing different minimizers/fitting software ba
 
-For help of how to use the command line/terminal, click the hyperlink corresponding to your operating system: [Windows](https://red-dot-geek.com/basic-windows-command-prompt-commands/), [macOS](http://newsourcemedia.com/blog/basic-terminal-commands/) and [Ubuntu](https://hackingpress.com/basic-linux-commands/)
+For help on how to use the command line/terminal, click the hyperlink corresponding to your operating system: 
+[Windows](https://red-dot-geek.com/basic-windows-command-prompt-commands/),
+[macOS](http://newsourcemedia.com/blog/basic-terminal-commands/) and
+[Ubuntu](https://hackingpress.com/basic-linux-commands/)
 
-1. Download this repository or clone it using [git](https://git-scm.com/):
-`git clone https://github.com/mantidproject/fitbenchmarking.git`
-2. Open up a terminal (command prompt) and go into the `fitbenchmarking` directory.
-3. Once you are in the right directory, type `python setup.py install`.
-4. Install mantid, platform specific instructions [here](https://github.com/mantidproject/fitbenchmarking/wiki/Installing-Mantid).
-5. Open the mantidpython terminal
-    * Ubuntu: in a normal terminal cd to /opt/Mantid/bin and run `./mantidpython`
-    * Windows: go to where you installed Mantid and search for a file called `mantidpython`
-    * macOS: in a normal terminal run `/Applications/MantidPlot.app/Contents/MacOS/mantidpython`
-#### For macOS only ####
-6. On the mantidpython terminal, run the following IPython command:
-    - `!/usr/bin/python -m pip install docutils==0.12 --user`
-
-The above step is done to ensure that the compatible version of docutils packages (version 0.12) is installed. 
-
-7. Finally, in this terminal, run `example_scripts/example_runScript_mantid.py`, located in the fitbenchmarking folder. This example script fit benchmarks Mantid using all the available minimizers. The resulting tables can be found in `example_scripts/results`.
+### Installing
 
-## FitBenchmarking Scipy
-The `example_runScripts.py` file is designed to benchmark minimizers supported by software/libraries that provide straightforward cross-platform Python install; as of know this mean SciPy (more details provided in the file itself).
-
-For this to work scipy version 0.17 or higher is needed (which includes needed [curve_fit](https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.curve_fit.html) support). **The Linux distributions we have tested against so far have all included scipy 0.17+ (0.17 is from Feb 2016).**
+1. Download this repository or clone it using [git](https://git-scm.com/): \
+    `git clone https://github.com/fitbenchmarking/fitbenchmarking.git`
+2. Open up a terminal (command prompt) and go into the `fitbenchmarking` directory.
+3. Once you are in the right directory, start the install with:\
+    `python setup.py install`
```

Comment:
I need to append `--user` to get this to work without needing sudo on Ubuntu

Reply:
Fixed